### DISCUSSION TITLE
How do I prevent a 404 with dynamic routes in NextJS 13?

### DISCUSSION THREAD

**Discussion by rwacarter** [vercel/next.js#46938] [Help]
### Summary

I have a Next13 app with an incredibly simple folder structure:
```
- app
  - users
    - [id]
      - page.tsx
  - ...
```
And a page.tsx with content similar to the following:
```
export default function UserPage({ params }) {
    return <div>{params.id}</div>
}
```
However, when I run next dev and go to /users/1, I get a 404. Is there something I'm missing about how dynamic routes work in NextJS 13?

**Comment by icyJoseph** [comment]
Hi,

Tested with next@13.2.3 and it works fine. What version are you using? Perhaps you have more routes?

**Comment by xllentech** [comment]
We are having similar issue, any version above 13.4.12 including 14+, dynamic routes get 404 with chunk load failed. App router, self-hosted using apache. Evething works as expected upto 13.4.12.
